test_scripts/Polices/appID_Management/ATF_P_TC_OnAppPermissionChanged_appRevoked_true.lua

```diff
@@ -0,0 +1,70 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+--     [ChangeRegistration]: appRevoked:true
+-- Description:
```

Comment:
In Description section you should use description like the one in the requirement itself: 
In case the <appID> app is currently registered and in any HMILevel and in result of PTU "<appID>" gets "null" policies, PoliciesManager must send OnAppPermissionChanged (appRevoked: true, appID) to HMI.